packages/odyssey-storybook/src/components/odyssey-mui/Button/Button.stories.tsx

```diff
@@ -107,4 +110,5 @@ export const IconOnly = Template.bind({});
 IconOnly.args = {
   children: undefined,
```

Comment:
We shouldn't need to define children as `undefined` right?

Just don't set a default value in the top-level args and define that in the default story. Icon-only could use a separate story template. The story is confusing to another developer otherwise.